scss/_patterns_logo-section.scss

```diff
@@ -15,6 +15,10 @@
   $logo-section-item-gap: 3rem;
   $logo-section-item-gap-dense: calc(0.5 * $logo-section-item-gap);
 
+  // TODO: Many styles for the dense variant within this block are overridden
+  // below; this should be separated into a "common" styles block for regular
+  // and dense variants and a block used solely for the regular variant,
+  // leaving the block at the bottom of the file for the dense variant
   .p-logo-section,
   .p-logo-section--dense {
```

Comment:
Add a comment here that we are deprecating this `// deprecated: use p-logo-section instead` (or something along these lines) - so we can later search for "deprecated" when preparing next major release.